### DISCUSSION TITLE
How to extract double column text?

### DISCUSSION THREAD

**Discussion by Flower-Wang1024** [py-pdf/pypdf#2982] [Q&A]
Hello, I now have a pdf document with a very regular structure, as shown in the figure, and I want to know how to extract the text in two columns instead of directly extracting it line by line.
![image](https://github.com/user-attachments/assets/7da769f9-365f-4647-9af2-af49800145dd)

**Comment by stefan6419846** [comment]
What have you tried so far? Does the layout mode suffice? Otherwise, you might want to have a look a the visitors, but without an actual PDF file and further analysis, there is not much we can further recommend here.

**Reply by Flower-Wang1024** [reply]
Thank you for your answer, I will try layout mode first.

**Reply by Flower-Wang1024** [reply]
[exampel1](https://pypdf.readthedocs.io/en/stable/user/extract-text.html#example-1-ignore-header-and-footer )why I copy the code and run it, it doesn't work., It prints blank.
![image](https://github.com/user-attachments/assets/7cf9a8f9-e019-4fe9-9e19-3c741b613139)

**Reply by stefan6419846** [reply]
See #2431 and the discussions in #2432.

TL;DR: You probably want to use `mult(cm, tm)` here instead of `cm`.
